tests/unit/core/api/test_search.js

```diff
@@ -0,0 +1,301 @@
+/* global window */
+import config from 'config';
+
+import search from 'core/api/search';
+import {
+  ADDON_TYPE_EXTENSION,
+  ADDON_TYPE_THEME,
+  CLIENT_APP_ANDROID,
+  CLIENT_APP_FIREFOX,
+} from 'core/constants';
+import { parsePage } from 'core/utils';
+import { createApiResponse } from 'tests/unit/core/api/helpers';
+import { dispatchSignInActions } from 'tests/unit/amo/helpers';
+import { unexpectedSuccess } from 'tests/unit/helpers';
+
+
+describe(__filename, () => {
+  let api;
+  let mockWindow;
+  const apiHost = config.get('apiHost');
+  const esrFirefoxUA = 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10.12; rv:52.2.1) Gecko/20100101 Firefox/52.2.1';
+  const newAndroidUA = 'Mozilla/5.0 (Android 4.4; Tablet; rv:57.0) Gecko/57.0 Firefox/57.0';
+  const newFirefoxForIOSUA = 'Mozilla/5.0 (iPad; CPU iPhone OS 8_3 like Mac OS X) AppleWebKit/600.1.4 (KHTML, like Gecko) FxiOS/1.0 Mobile/12F69 Safari/600.1.4';
+  const newFirefoxUA = 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10.12; rv:57.0) Gecko/20100101 Firefox/57.1';
```

Comment:
Yeah good point. I'll actually just move them to the `userAgent` helpers. I'd like to improve that helper so it has useful keys (eg `userAgent.firefox57`) or something but that's for the future 😉 

I'll just move them to the existing helper so everything's at least in one place.